span_test.go

```diff
@@ -28,6 +28,17 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+type fakeBaggageRestrictionManager struct {
```

Comment:
you could actually move this type to real code if you call it `simpleB..R..M..`, instead of storing the map on the main manager directly. Just a thought.